pyproject.toml

```diff
@@ -4,3 +4,68 @@ line-length = 79
 [tool.isort]
 profile = "black"
 skip = ["flowpipe/__init__.py"]
+
+[tool.pylint."MESSAGES CONTROL"]
+disable = [
+    "R0801",
+    "too-many-function-args",
+    "too-many-locals",
+    "too-many-arguments",
+    "too-few-public-methods",
+    "too-many-instance-attributes",
+    "too-many-public-methods",
+    # Omitted because of Python2.7 support:
+    "consider-using-f-string",
+    "super-with-arguments",
+    "useless-object-inheritance",
+    "raise-missing-from"
+]
+
+[tool.poetry]
+name = "Flowpipe"
+version = "0.10.0"
+description = "A lightweight framework for flow-based programming in python."
+authors = ["Paul Schweizer <paulschweizer@gmx.net>"]
+license = "MIT"
+readme = "README.md"
+repository = "https://github.com/PaulSchweizer/flowpipe"
+documentation = "https://flowpipe.readthedocs.io/en/latest/"
+classifiers = [
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 2.6",
+    "Programming Language :: Python :: 2.7",
+    "Programming Language :: Python :: 3.4",
+    "Programming Language :: Python :: 3.5",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
```

Comment:
What about py 3.8 - 3.10, should we include them?

Reply:
Yes, also done